File: data/code_pairs/fortran/885.f90
module Members
  implicit none

  type :: HasMembers
    integer :: kind_member
    integer :: len_member
    integer :: member
  contains
    procedure :: MyMethod
  end type HasMembers

contains

  subroutine MySubroutine(arg)
    integer :: arg
    print *, 'MySubroutine called with arg =', arg
  end subroutine MySubroutine
  
  subroutine MyMethod(this)
    class(HasMembers), intent(in) :: this
    print *, 'Member value:', this%member
  end subroutine MyMethod

end module Members

program test
  use Members
  implicit none
  
  type(HasMembers) :: obj
  
  obj%kind_member = 1
  obj%len_member = 10
  obj%member = 5
  
  call obj%MyMethod()
  
  call MySubroutine(10)
  
end program test

File: data/code_pairs/cpp/885.cpp
#include <iostream>

namespace Members {

    class HasMembers {
    public:
        int kind_member;
        int len_member;
        int member;

        HasMembers(int kind, int len, int mem) : kind_member(kind), len_member(len), member(mem) {}

        void MyMethod() const {
            std::cout << "Member value: " << member << std::endl;
        }
    };

    void MySubroutine(int arg) {
        std::cout << "MySubroutine called with arg = " << arg << std::endl;
    }

}